py/circuitpy_mpconfig.mk

```diff
@@ -65,6 +65,9 @@ CFLAGS += -DCIRCUITPY_ALARM=$(CIRCUITPY_ALARM)
 CIRCUITPY_ANALOGIO ?= 1
 CFLAGS += -DCIRCUITPY_ANALOGIO=$(CIRCUITPY_ANALOGIO)
 
+CIRCUITPY_ADCBUFFER ?= 1
```

Comment:
Set this to `?= 0' and it will be off by default in other ports. You already set it to 1 for the RP2040 port.
I did a quick build and the ATMEL builds worked for me after that.